docs/benchmark.md

```diff
@@ -42,13 +42,13 @@ We provide the training log based on which we calculate the average iter time, w
 
 | Model  |input| io backend | batch size | MMAction2 (s/iter) | MMAction (s/iter) | Temporal-Shift-Module (s/iter) | PySlowFast (s/iter) |
 | :--- | :---------------:|:---------------:| :---------------:| :---------------:  | :--------------------: | :----------------------------: | :-----------------: |
-| [TSN](/configs/recognition/tsn/tsn_r50_1x1x3_100e_kinetics400_rgb.py)| 256p rawframes |MemCached| 32x8|**[TODO]()** | [TODO]() | [TODO]() | x |
+| [TSN](/configs/recognition/tsn/tsn_r50_1x1x3_100e_kinetics400_rgb.py)| 256p rawframes |MemCached| 32x8|**[~~1.01(8339M)~~]()** | [TODO]() | [0.42(10733M)]() | x |
```

Comment:
interesting

Reply:
It seems something went wrong. Maybe some redundant operation? like `resize`? And In my experience, you can try `fast` option to speed it up, it is also effective for 256p.

Reply:
1s for 256p does not make sense